roles/serviceassurance/tasks/component_events_smartgateway.yml

```diff
@@ -1,3 +1,29 @@
+#- name: Check if elasticsearches.elasticsearch.k8s.elastic.co CRD is available
+#  k8s_facts:
+#    api_version: 'apiextensions.k8s.io/v1beta1'
+#    kind: 'customresourcedefinition'
+#    name: 'elasticsearches.elasticsearch.k8s.elastic.co'
+#  register: elasticsearch_crd
+#
+#- debug:
+#    var: elasticsearch_crd
+
+- name: Lookup ElasticSearch BasicAuth
+  k8s_facts:
+    api_version: v1
+    kind: Secret
+    namespace: '{{ meta.namespace }}'
+    name: 'elasticsearch-es-elastic-user'
+  register: elasticsearch_es_elastic_user
+
+- name: Filter out ElasticSearch password for BasicAuth
+  set_fact:
+    elastic_pass: "{{ elasticsearch_es_elastic_user | json_query('resources[].data.elastic') | b64decode }}"
```

Comment:
Adding `pip3 install --user jmespath` to `Dockerfile` satisfies dep for this command.

Reply:
Name is dynamic. Need to fix.

Reply:
Actually I am wrong. This is fine.